src/api/environment.cc

```diff
@@ -498,4 +498,32 @@ uv_loop_t* GetCurrentEventLoop(Isolate* isolate) {
   return env->event_loop();
 }
 
+void AddLinkedBinding(Environment* env, const node_module& mod) {
```

Comment:
how come this isn't `Environment::AddLinkedBinding(const node_module& mod)`?

Reply:
@devsnek It’s a public API, and we currently don’t export any variants of our internal classes in `node.h`